queue_services/entity-filer/src/entity_filer/services/bootstrap.py

```diff
@@ -0,0 +1,275 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: remove this file once filer refers to common module
+"""This is a service to bootstrap the Incorporation Application process."""
```

Comment:
Small thing, is this accurate? Maybe Incorporation/Registration? Same for line 33. Maybe IA/registration.

Reply:
You're right. But I tend to make no changes in this file. It's just a copy from legal-api, and we will finally sync a version in a common module (similar with common data models) so that other queue services can use it too.

Reply:
Got it. Sounds good, thanks! Resolving.